### DISCUSSION TITLE
LANGUAGE: Vietnamese for Publii

### DISCUSSION THREAD

**Discussion by ManhTuan689** [GetPublii/Publii#1888] [General]
I have just completed the Vietnamese translation for Publii for personal use. However, I want to contribute to making Publii accessible to everyone, everywhere, so I have decided to share this Vietnamese language package for free.

You can download: [vi-VI.zip](https://github.com/user-attachments/files/18205605/vi-VI.zip)

If you have any suggestions or edits, feel free to contact me via:

Email: ksnguyenmanhtuan@gmail.com
Facebook: https://www.facebook.com/TuanxPlus
Note: This Vietnamese localization is only 80% complete, as some keys are not included in the basic language package.

**Comment by dziudek** [comment]
@ManhTuan689 - you can create pull request with your translation here: https://github.com/GetPublii/Publii-ui-locales :)
